datafusion/physical-expr/src/aggregate/mod.rs

```diff
@@ -77,6 +79,18 @@ pub trait AggregateExpr: Send + Sync + Debug {
     fn name(&self) -> &str {
         "AggregateExpr: default name"
     }
+
+    /// If the aggregate expression is supported by row format
```

Comment:
This approach of adding two new functions to the `Aggregate` trait has the very nice property that it is backwards compatible. However, I wonder if we can clarify:
1. Do aggregates need to implement `accumulator` if they also implement `create_accumulator_v2`?
2. What is the benefit of creating accumulator v2 (it is faster?)